views/View.Section.class.php

```diff
@@ -34,11 +34,16 @@ public function render() {
 			return;
 		}
 
+		$settings = get_option( 'food-and-drink-menu-settings' );
+		$show_empty = $settings['fdm-show-empty-sections'];
```

Comment:
`$settings['fdm-show-empty-sections']` may not exist if no settings have been saved. Try:

```
$show_empty = isset($settings['fdm-show-empty-sections']) && !empty($settings['fdm-show-empty-sections']);
```